luigi/contrib/ecs.py

```diff
@@ -58,7 +58,11 @@
 
 try:
     import boto3
-    client = boto3.client('ecs')
+    try:
+        import botocore.exceptions
+        client = boto3.client('ecs')
+    except botocore.exceptions.NoRegionError:
+        logger.warning('Your AWS config is missing Region information, ECSTask requires a working config.')
```

Comment:
it's necessary in order to make these unit tests work, because ECSTask does not use boto3 in an extensible manner.

Basically, the ecs_test module appears to have been written for someone's personal testing instead of for Travis CI. When the boto3 package is available for testing in Travis (since I added it to tox.ini), the ecs_test module is suddenly activated by its presence. It then proceeds to fail because boto3.client will always fail when no local AWS configuration is available. 

Since boto3 is now available, I have contributed a fix for their tests so that their tests won't fail the build when my tests are present.